File: tsdsfe/scripts/tsdsfe.m
% Enter the following commands at the MATLAB/Octave command line prompt.
% Data will be placed in matrix named D and column labels in cell array L.
% __COMMENT__

SVR = '__SERVER__';
QS  = '__QUERYSTRING__';
tmp = [SVR,'?',QS];
fprintf('tsdsfe.m: Downloading %s\n',tmp);
[Ds,s]  = urlread(tmp); 
if (s == 0)
    fprintf('tsdsfe.m: Error when downloading.  Check server status at __STATUS__\n');
end
D   = str2num(Ds);
L   = {'Year','Month','Day','Hour','Minute','Second',__LABELS__};

fprintf('tsdsfe.m: Data matrix D [%d,%d] has column labels: ',size(D,1),size(D,2));
tmp = sprintf('%s, ',L{:});fprintf('%s\n',tmp(1:end-2));
fprintf('tsdsfe.m: First timestamp: %s\n',datestr(D(1,1:6),31));
fprintf('tsdsfe.m: Last timestamp:  %s\n',datestr(D(end,1:6),31));

File: tsdsfe/scripts/tsdsfe.py
# Tested in Python 2.7.
# __COMMENT__
import csv
import urllib2

url = '__SERVER__?__QUERYSTRING__'
try:
	response = urllib2.urlopen(url)
except:
	print 'tsdsfe.py: Error when downloading.  Check server status at __STATUS__';
	import sys
	sys.exit()

cr = csv.reader(response, delimiter=' ')
D = [row for row in cr]
L = ['Year','Month','Day','Hour','Minute','Second',__LABELS__]

print "Labels and first/last line."
print (D)
print D[0]
print D[-1]